vipps-login-api.md

```diff
@@ -1441,6 +1441,174 @@ In addition to the responses defined by [the standard](https://openid.net/specs/
+    Content-Type: application/json;charset=UTF-8
+    {
+      "error": "authorization_pending",
+      "error_description": "The authorization request is still pending"
+    }
+    ```
+
+   The merchant should keep polling when it receives `authorization_pending` in the error response. After the user completes the login in the app, the `token` endpoint will give a successful response similar to the following example:
+
+    ```http
+    HTTP/1.1 200 OK
+     Content-Type: application/json;charset=UTF-8
+
+    {
+      "access_token": "ciba.W_IfBcSr-askdjhsakjhd",
+      "token_type": "Bearer",
+      "expires_in": 300,
+      "id_token": "eyaksjdhksajhdjkashdjksadjnn91283hedhn.eyasdkjhaskjdhskajhdkjhasdkjhaskjhdwqiuh"
+    }
+    ```
+
+3. The merchant must do a GET to the `userinfo` endpoint with the header: Authorization: Bearer {access_token}, using the access_token retrieved in step 2.
+
+   For details see [Userinfo request](#userinfo).
+
+   Example request:
+
+    ```http
+    GET https://api.vipps.no/vipps-userinfo-api/userinfo
+    Authorization: Bearer ciba.W_IfBcSr-askdjhsakjhd
+    ```
+
+   Example response:
+
+    ```http
+    HTTP/1.1 200 OK
+
+    {
+      "family_name": "Heyerdahl",
+      "given_name": "Tor Fos",
+      "name": "Tor Fos Heyerdahl",
+      "other_addresses": [],
+      "sid": "qwieuhwqiuhdiuwqh",
+      "sub": "f350ef33-22e2-47d0-9f47-12345667"
+      "delegatedConsents" : {
+        "language" : "EN",
+        "heading" : "Give consent",
+        "text" : "oidc-testclient wants to send you relevant offers through their digital channels.",
+        " termsDescription" : "By confirming, you accept oidc-testclient's terms of membership. They are responsible for processing the consents. You can withdraw them at any time on their website.",
+        "confirmConsentButtonText" : "Confirm",
+        "links" : {
+          "termsLinkText" : "Read the terms of use.",
+          "termsLinkUrl" : "https://www.vipps.no/vilkar/vilkar-privat/",
+          "privacyStatementLinkText" : "Read the privacy policy.",
+          "privacyStatementLinkUrl" : "https://www.vipps.no/vilkar/cookie-og-personvern/"
+        },
+        "timeOfConsent" : "2022-11-23T11:40:13Z",
+        "consents" : [ {
+          "id" : "email",
+          "accepted" : true,
+          "required" : true,
+          "textDisplayedToUser" : "Receive offers via email"
+        }, {
+          "id" : "sms",
+          "accepted" : true,
+          "required" : false,
+          "textDisplayedToUser" : "Receive offers via SMS"
+        }, {
+          "id" : "digital",
+          "accepted" : true,
+          "required" : false,
+          "textDisplayedToUser" : "I would like to receive digital marketing"
+        }, {
+          "id" : "personal",
+          "accepted" : true,
+          "required" : false,
+          "textDisplayedToUser" : "Get customized offers"
+        } ]
+      }
+    }
+    ```
+
+#### Authentication request
```

Comment:
On these little sections here, we should have a space after the title and periods at the ends of the sentences. :-)